coverage/clover.xml

```diff
@@ -0,0 +1,59 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
maybe we can ignore `coverage` folder for the moment

Reply:
Done